Packs/Darktrace/Integrations/Darktrace/Darktrace_test.py

```diff
@@ -361,3 +361,102 @@ def test_get_entity_details(requests_mock):
 
     assert integration_response.outputs == expected_response
     assert integration_response.outputs_prefix == 'Darktrace.EntityDetails'
+
+
+def test_get_modelbreach_details(mocker):
+    """Tests the list-similar-devices command function.
+    Configures requests_mock instance to generate the appropriate
+    get_alert API response, loaded from a local JSON file. Checks
+    the output of the command function with the expected output.
+    """
```

Comment:
Seems this is taken from another test. Please edit these docs (also on the following).